app/src/interfaces/_system/system-display-template/system-display-template.vue

```diff
@@ -76,20 +79,34 @@ const injectValue = computed(() => {
 
 	return { fields: [fakeVersionField] };
 });
+
+const tree = computed(() => {
+	if (props.fields) {
+		return { list: props.fields };
+	}
+
+	if (collection.value === null) {
+		return null;
+	}
+
+	const { treeList, loadFieldRelations } = useFieldTree(collection, injectValue);
```

Comment:
Can we move the composable out of the computable? Composable should generally be placed on top-level and there's no direct dependency here

Reply:
Done :+1:

Reply:
Agreed, although I believe that was done due to `useFieldTree` erroring if a collection is not provided.